index.bs

```diff
@@ -510,10 +510,18 @@ The sample code for generating and registering a new key follows:
       // This Relying Party will accept either an EdDSA, ES256 or RS256 credential, but
       // prefers an EdDSA credential.
       pubKeyCredParams: [
+        {
+          type: "public-key",
+          alg: -50 // "Ed25519" as registered in the IANA COSE Algorithms registry
```

Comment:
This is assigned to ML-DSA-87 in IANA https://www.iana.org/assignments/cose/cose.xhtml

Reply:
Thanks for catching the conflict! Draft 12 changed the assignment request to -19: https://www.ietf.org/archive/id/draft-ietf-jose-fully-specified-algorithms-12.html#name-edwards-curve-digital-signa . Fixed in c129a055c23e97b01caf513aa130a48d3904afe2.